src/SDKs/Azure.Base/data-plane/Azure.Base/Http/HttpPipelineResponse.cs

```diff
@@ -9,7 +9,7 @@ namespace Azure.Base.Http
 {
     public abstract class HttpPipelineResponse: IDisposable
     {
-        public abstract int Status { get; }
+        public abstract int Status { get; set; }
```

Comment:
I am not thrilled that now we need public set. Can we not make it public?

Reply:
It's not required for this PR so I'll undo it but I think that response properties should be setable by policies the same way requests are.